packages/react-microlink/src/index.js

```diff
@@ -97,7 +104,7 @@ Microlink.propTypes = {
   apiKey: PropTypes.string,
   contrast: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
   image: PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)]),
-  prerender: PropTypes.bool,
+  prerender: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
```

Comment:
How about
```
prerender: PropTypes.oneOfType(PropTypes.oneOf(['auto']), PropTypes.bool]),
```
(not 100% sure if that's the correct syntax)

Reply:
Looks like it's valid 👍